_posts/2020-05-15-issue-2.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+title: 'Issue 2: Who calls the shots?'
+category: Volume 2
+image:
+    file: vol-2-issue-2-header.png
+    alt: "Screenshot of a tweet sent by Elon Musk announcing the re-opening of a Tesla factory, contra local safety regulations"
+    caption: "Elon Musk acting as if he's some sort of civil rights hero for defying regulations meant to keep workers safe"
+    source_link: "https://twitter.com/elonmusk/status/1259945593805221891"
+    half_width: false
+preview: "Elon Musk gets to call the shots. He shouldn't."
+---
+
+Who calls the shots at Tesla Motors? Is it the workers who collectively build all the products that generate the company's wealth? No, it's the company's multi-billionaire CEO, who recently threw a fit on Twitter over health and safety regulations that would delay resuming production. How does he manage to get away with that?
```

Comment:
minor clarification to highlight the cause of Musk's tantrum: "who recently threw a fit on Twitter over COVID-19 shelter-in-place orders that would delay resuming production."

Reply:
done